app/views/cart_pursuits/_edit_pursuit.html.haml

```diff
@@ -1,17 +1,26 @@
-.cart
-  - @pursuits.each do |pursuit|
-    .row
-      .col.s12.m7
-        .card-panel.teal.white-text
-          .pursuit_card
-            = render pursuit
+.cart.white
+  %table
+    - @pursuits.each do |pursuit|
+      .pursuit_card
+        %tr
+          .row
+            .col.s3
+              %img.responsive-img(src="http://www.fantasiavillas.com/wp-content/uploads/2012/03/caribbean-grenada-calivigny-island-8.jpg")
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))<br>Line is too long. [138/80]